lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -207,14 +251,21 @@ def add_debt_repayment(doc, options)
       end
 
       def add_payment_method(doc, payment_method, options)
-        if payment_method.is_a?(String)
+        if payment_method.is_a?(String) || payment_method.is_a?(Numeric)
           doc.token do
             doc.litleToken(payment_method)
           end
         elsif payment_method.respond_to?(:track_data) && payment_method.track_data.present?
           doc.card do
             doc.track(payment_method.track_data)
           end
+        elsif check?(payment_method)
+          doc.echeck do
+            doc.accType(payment_method.account_type)
+            doc.accNum(payment_method.account_number)
```

Comment:
We'll want to add this field to the scrub method

Reply:
good catch.

Reply:
And the scrub tests too, preferably.

Reply:
👍 check and check